src/NHibernate.Test/NHSpecificTest/NH2188/AppDomainWithMultipleSearchPath.cs

```diff
@@ -24,6 +24,10 @@ public void WhenSerchInMultiplePathsThenNotThrows()
 			string binPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "bin");
 			var expected = Path.Combine(binPath, Configuration.DefaultHibernateCfgFileName);
 
+			// Test uses an obsolete .Net feature, changing the private path of an ongoing AppDomain.
+			// If that feature is dropped, the tests will likely need to be dropped too. (Or heavily changed
+			// for running in a newly created AppDomain.)
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
I have not found a way to do that with non-obsolete methods and without spawning a new app domain for the test. It seems the used methods have been obsoleted precisely because they were altering private path for already created app domain, which the .Net designers seem to advise against.